lib/core/components/MantraForm.js

```diff
@@ -0,0 +1,63 @@
+import get from 'lodash/get';
+import cloneDeep from 'lodash/cloneDeep';
+import MantraComponent from './MantraComponent';
+import MantraData from './../MantraData';
+import { error } from '../../utils';
+
+/**
+ * Class representing a form component configuration.
+ * @extends MantraComponent
+ */
+export default class MantraForm extends MantraComponent {
+    /**
+     * Creates a MantraForm configuration.
+     * @param {object} payload - Contains the value of the MantraForm properties.
+     */
+    constructor({ name, fields }) {
+        super({ name, mixin: 'MantraForm' });
+        this.fields = fields;
+    };
+
+    /**
+     * Get the config of a MantraForm component.
+     * @param {object} payload - Contains the path and config for MantraData creation.
+     * @returns {MantraData} - The MantraForm component config.
+     */
+    static _getConfig({ path }) {
+        const config = new MantraData({ path, baseClass: MantraForm });
```

Comment:
We decided to keep the classes used as Strategy (for now) to be resolved in the target class (in this case MantraData) with closures that avoid the circular reference to be triggered at compilation time.

This translate into the baseClass param to be just a String 'MantraForm' and the actual resolving of the MantraForm as class occurs at the MantraData class

Reply:
Yes! Thank you for reminding me 'cause I forgot this is a "must do" change.

Will apply the changes right away